### DISCUSSION TITLE
You cannot apply because you do not have a school email address.

### DISCUSSION THREAD

**Discussion by jinhyeon-dev** [community/community#112084] [GitHub Education]
### Hi everyone,

I'm trying to receive student support, but when selecting an email, it says that only school emails are accepted, but my email is not accepted. Can’t I just use my proof of enrollment?

**Comment by probablynotnmp** [comment]
Certainly, you have the option to use any proof of enrollment, such as your University ID Card, enrollment letter, or transcript if you do not get a school mail address.

**Comment by SebastienXia** [comment]
Hi, @jinhyeon-dev 
Personally I gave my personal email address and I had access to the Benefits.
